hist/histv7/inc/ROOT/RAxis.hxx

```diff
@@ -438,17 +532,6 @@ public:
    int GetBinIndexForLowEdge(double x) const noexcept;
 };
 
-/// Equality-compare two RAxisEquidistant.
-inline bool operator==(const RAxisEquidistant &lhs, const RAxisEquidistant &rhs) noexcept
-{
-   return lhs.GetNBins() == rhs.GetNBins() && lhs.GetMinimum() == rhs.GetMinimum() &&
-          lhs.GetInverseBinWidth() == rhs.GetInverseBinWidth();
-}
-inline bool operator!=(const RAxisEquidistant &lhs, const RAxisEquidistant &rhs) noexcept
-{
-   return !(lhs == rhs);
-}
-
```

Comment:
I don't think it is a good idea to have an equality operator which only compares some properties of RAxisEquidistant, but not all of them. In my opinion, operator overloading should only be used when there is only one obvious meaning to the operator at hand, which in this case would be the somewhat useless meaning of comparing every property.